iac/terraform/aws/api_gw.tf

```diff
@@ -81,11 +81,11 @@ resource "aws_lambda_permission" "lambda_invoke_permission" {
 
 #resource "aws_api_gateway_domain_name" "api_gw_domain_name" {
 #  domain_name     = "${var.environment}.${var.bloodconnect_domain}"
-#}
-#
-#resource "aws_api_gateway_base_path_mapping" "api_mapping" {
+# }
+
+# resource "aws_api_gateway_base_path_mapping" "api_mapping" {
 #  api_id      = aws_api_gateway_rest_api.rest_api.id
 #  stage_name  = aws_api_gateway_deployment.api_deployment.stage_name
```

Comment:
same as above

Reply:
I thought to keep it as it's not written by me.